### ISSUE TITLE
Version 1.2.0 How do I set up a Trial Quest?

### ISSUE THREAD

**Issue by Sarasena** [Masuzu/ZooeyBot#34]
1.) I use Viramate and enabled it in ini file.
2.) I left LuaScript to empty since I just want my characters to attack. I didn't want to use any scripts for testing.
3.) I use Basic Mode. Since SlaveMode.Active, EventMode.Enabled and Debug.TrialBattleMode are all set to false.
4.) I successfully calibrated Zooeybot. It says good to go. But, it does not ask me to navigate to a page. Just nothing. When I click on a quest, nothing happens too. So I tried going to the quest itself and chose a summon and waited...Nothing happens. 
5.) I restarted a couple of times + reloading the ID as instructed. Most of the time, Zooey is stuck on calibration mode and it's rarely that I can make it run to "Good to go."

I reread your instructions multiple times, watched your vids and still have nothing. I am trying to farm this Aurora Trial quest. I also started with 232 AP just in case.

I'm at loss on where to go from here and since my trial already expired (7hrs is so short btw) I have to sub and I still can't make it work. I've been trying for the past 3 days since the early version. Please help.

**Comment by Kentiah** [comment]
Try disabling viramate in the ini instead, I believe if you use viramate you need to enable hotkeys in viramate as well. I personally use viramate, but I run zooeybot as if I didn't.

**Comment by Masuzu** [comment]
Hi,

You are stuck in the calibration:
It it does not display a message saying "Vertical offset: XXX", then I am sorry but it is a blatant proof of mis scaling issue on your side: https://github.com/Masuzu/ZooeyBot/wiki/FAQ#i-have-scaling-issues-again
This and make sure to have the "Home" button visible in the top left hand corner.

```
4.) I successfully calibrated Zooeybot. It says good to go. But, it does not ask me to navigate to a page. Just nothing. When I click on a quest, nothing happens too. So I tried going to the quest itself and chose a summon and waited...Nothing happens.
```

**After** the calibration is done, then you will have to navigate to http://game.granbluefantasy.jp/#quest. Refresh the page if you are already on this screen and that's it.

**Comment by Sarasena** [comment]
Hi Masuzu,
I do not have scaling issues. In fact here's an attached file:
![image](https://cloud.githubusercontent.com/assets/28635741/25985567/b8aa1176-371d-11e7-9ce0-21ce6e9a3934.png)

However, I managed to make it work after restarting Google Chrome and not playing GBF the first time. And like I said, I already refreshed the page multiple times on quest page to see if it will work. I think there is a possible issue if you are using Multiple Profiles (I use one for GBF and one for work to not interfere with each other).
